src/encoded/types/gene.py

```diff
@@ -0,0 +1,81 @@
+from snovault import (
+    calculated_property,
+    collection,
+    load_schema,
+)
+from .base import (
+    SharedItem,
+    paths_filtered_by_status,
+)
+from pyramid.traversal import (
+    find_root,
+)
+
+
+@collection(
+    name='genes',
+    unique_key='gene:name',
+    properties={
+        'title': 'Genes',
+        'description': 'Listing of ENCODE genes',
```

Comment:
"ENCODE" genes may be not quite right...  Maybe this should just be "Listing of genes"

Reply:
Corrected in ENCD-3997-modification-1 commit.